### ISSUE TITLE
could not find module 'SwiftyTesseract' for target 'armv7-apple-ios'; found: arm64, arm64-apple-ios

### ISSUE THREAD

**Issue by sikandernoori** [arrrrny/tesseract_ocr#34]
User@MacBook-Pro-1892 ocr_demo % flutter build ios
Building com.packagename.ocrDemo for device (ios-release)...
Automatically signing iOS for device deployment using specified development team in Xcode project: xxxxxx
Running pod install...                                              6.0s
Running Xcode build...                                                  
Xcode build done.                                           91.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/xxxx/.pub-cache/hosted/pub.dartlang.org/tesseract_ocr-0.3.1/ios/Classes/SwiftTesseractOcrPlugin.swift:3:
    8: error: could not find module 'SwiftyTesseract' for target 'armv7-apple-ios'; found: arm64, arm64-apple-ios
    import SwiftyTesseract
           ^
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported
    deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Encountered error while building for device.

**Comment by Daniel-Egorov** [comment]
I am getting a similar error
Could not find module 'SwiftyTesseract' for target 'arm64-apple-ios-simulator'; found: x86_64-apple-ios-simulator
were you able to fix this?

**Comment by sikandernoori** [comment]
@Daniel-Egorov Yes, there were some issues with this Plugin so I forked and created a new one with some changes.

https://pub.dev/packages/tesseract

But this one is also outdated as I am not maintaining it, but it should get you around this error
